src/lib/viewers/iframe/IFrameLoader.js

```diff
@@ -22,6 +23,20 @@ class IFrameLoader extends AssetLoader {
         super();
         this.viewers = VIEWERS;
     }
+
+    /**
+     * @inheritdoc
+     */
+    determineViewer(file, disabledViewers = [], viewerOptions = {}) {
+        const isDicomFile = file.name === 'Dicom.boxdicom' || file.extension === 'boxdicom';
+        const openWithAmbraEnabled = getProp(viewerOptions, 'IFrame.openWithAmbra');
+        // The IFrame viewer is disabled when the file is a Boxdicom file and Open_with_Ambra FF is enabled
+        if (openWithAmbraEnabled && isDicomFile) {
+            disabledViewers.push('IFrame');
+        }
+
+        return super.determineViewer(file, disabledViewers);
```

Comment:
Should we pass `viewerOptions` in here as the third argument?

Reply:
It's not needed in this case since we are only disabling the IFrame viewer. However from my understanding if we were to have viewer options that modifies logic in both the IFrame loader and Asset loader then we would.

Reply:
I was mainly thinking about maintaining consistency with what the base class expects, but it looks like many other viewers fail to pass `viewerOptions` down, as well. Not a big deal in that case.